### DISCUSSION TITLE
如何在数据少的时候撑开table的高度

### DISCUSSION THREAD

**Discussion by wartim** [ant-design/ant-design#31959] [Q&A]
给table设置了最小高度，防止破坏布局
#card1 .ant-table {
min-height: 307px;
}
但现在的问题是如果数据多会出现滚动条看起来挺好，但数据少的时候 tbody高度怎么固定填充为最大，而且footer也移动到上面去了，这样就很丑
希望table整体保持一个固定高度，而且数据区高度自适应为最大可用区域
![1111111](https://user-images.githubusercontent.com/6902982/131248202-fbd540b8-4965-42fb-a09f-ba85c5fd448d.png)

**Comment by GhostLee** [comment]
顶一下   我也遇到这个问题了

**Comment by wangchao18654941093** [comment]
相同的问题

**Comment by icewind4096** [comment]
同求，个人觉得这个是基本需求吧

**Comment by alienzhangyw** [comment]
#37359
